skills/declarative/calendarSkill/runtime/customaction/Actions/MSGraph/CheckAvailability.cs

```diff
@@ -20,9 +19,9 @@ namespace Microsoft.BotFramework.Composer.CustomAction.Actions.MSGraph
     /// This has proven to be more consistent that LUIS datetime recognition by allowing the use of the 
     /// user's current timezone time as a relative datetime rather than a mix of absolute times and UTC times.
     /// </summary>
+    [MsGraphCustomActionRegistration(CheckAvailability.DeclarativeType)]
```

Comment:
This doesn't look like its using the same approach to setting the DeclarativeType as the other actions